tslint.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": [
+    "italia-tslint-rules/basic",
+    "italia-tslint-rules/react"
+  ],
+  "jsRules": {},
+  "rules": {
+    "jsx-no-lambda": false,
+    "jsx-no-multiline-js": false,
+    "no-console": false
```

you should enable `no-any`:

```suggestion
    "no-console": false,
    "no-any": true
```